docs/site/RBAC-with-authorization.md

```diff
@@ -47,4 +47,4 @@ The architecture diagram is:
 The example is created in repository
 [access-control-migration](https://github.com/strongloop/loopback-next/tree/master/examples/access-control-migration),
 and a tutorial of building it from scratch can be found in
-[access control example migration](./migration/auth/example.md).
+[access control example migration](https://loopback.io/doc/en/lb4/migration-auth-access-control-example.html).
```

Comment:
With the rename, does the relative path work?

Reply:
> With the rename, does the relative path work?

I agree, let's try `migration-auth-access-control-example.md` first.